### DISCUSSION TITLE
IsSystemList available?

### DISCUSSION THREAD

**Discussion by LegendaryB** [pnp/pnpcore#589] [Q&A] [answered]
Hey,

I was wondering if we can somehow retrieve this property? I did not find a way to do that with PnP yet? 
The idea came from this post on stackoverflow: https://stackoverflow.com/questions/37704813/sharepoint-csom-how-to-filter-out-system-default-document-libraries

**Comment by jansenbe** [comment, marked as answer]
@LegendaryB : now it is :-) See https://github.com/pnp/pnpcore/commit/46c1d7bd08283bfa50d11dbf1475c5fb6a4becec. Will be part of the next nightly build

**Reply by LegendaryB** [reply]
As fast as always 👍
